buffalo/cmd/dev.go

```diff
@@ -91,6 +91,13 @@ func startWebpack(ctx context.Context) error {
 	}
 
 	cmd := exec.CommandContext(ctx, webpack.BinPath, "--watch")
+	if app.WithYarn {
+		if found, err := YarnScriptExists("buffalo:dev"); err != nil {
+			return errors.WithStack(err)
+		} else if found {
```

Comment:
I feel like this can be collapsed to remove the else if. I had to read it a few times to understand the meaning of it.